### ISSUE TITLE
Question on T5 NLG

### ISSUE THREAD

**Issue by AtheerAlgherairy** [ConvLab/ConvLab-3#187]
I have a question regarding the training loss for T5 NLG. If we do not set 'metric_for_best_model' to 'bleu,' as shown in the picture below, is it automatically set to 'loss'? What is the best practice for training T5 NLG?

![Screen Shot 1445-07-10 at 11 47 57 AM](https://github.com/ConvLab/ConvLab-3/assets/7421065/a9072de1-6750-46fd-a94f-c91a3f414f10)

**Comment by zqwerty** [comment]
Yes, the default metric is loss. According to some NLG studies/practices, continually training the model after achieving the lowest validation loss can still improve metrics like BLEU.

**Comment by AtheerAlgherairy** [comment]
Thanks..

**Comment by AtheerAlgherairy** [comment]
Hi.. I used T5-base for NLG.. I got the following results..


![Screen Shot 1445-08-08 at 2 02 15 PM](https://github.com/ConvLab/ConvLab-3/assets/7421065/e4c90336-47a5-4762-b2d2-c88cb2836fe7)

![Screen Shot 1445-08-08 at 2 03 08 PM](https://github.com/ConvLab/ConvLab-3/assets/7421065/35484c42-ae56-43cd-afd0-39b1c58589c1)


Does the NLG evaluation calculate SER (Slot error rate) ?

**Comment by zqwerty** [comment]
Sorry for the late reply. Does "err" mean slot error rate? Maybe you could try some pre-training?
